assimilation_code/modules/assimilation/filter_mod.dopplerfold.f90

```diff
@@ -2344,9 +2357,8 @@ function count_state_ens_copies(ens_size, post_inflate, prior_inflate) result(nu
 !  then we need an extra copy to hold (save) the prior ensemble spread
 !   ENS_SD_COPY will be overwritten with the posterior spread before
 !   applying the inflation algorithm; hence we must save the prior ensemble spread in a different copy
-if ( inf_flavor(2) == 4 ) then ! CSS
-   SPARE_PRIOR_SPREAD = next_copy_number(cnum)
-endif 
+
+if (do_rtps_inflate(post_inflate)) SPARE_PRIOR_SPREAD = next_copy_number(cnum)
```

Comment:
This is the part that is not working in filter_mod.f90 ... and I don't know why ... When filter_mod.f90 is understood, this block may or may not need to be fixed.